MultiSource/Benchmarks/DOE-ProxyApps-C++/CLAMR/mesh.cpp

```diff
@@ -2760,7 +2760,12 @@ void Mesh::rezone_all(int icount, int jcount, vector<int> mpot, int have_state,
    mesh_memory.memory_swap(&level, &level_old);
 
    index.clear();
-   index.resize(new_ncells);
+   // allocate to ncells (oldsize) because index is a map from old cells to new
+   // cells loop below is for (ic = 0; ic < ncells; ic++) and within it
```

Comment:
```suggestion
   // cells. Loop below is for (ic = 0; ic < ncells; ic++) and within it
```

Though I don't think you really need any part of this comment apart from the first sentence. There is no need to describe a historical bug -- that's for the commit message, not the code.